src/components/DescriptionEditor.tsx

```diff
@@ -3,40 +3,54 @@ import { Fab, TextField, Typography } from '@mui/material';
 import EditIcon from '@mui/icons-material/Edit';
 import SaveIcon from '@mui/icons-material/Save';
 
+const defaultProps = {
+  label: null,
+  levelValue: null,
+};
+
 interface DescriptionEditorProps {
+  label?: string;
```

Comment:
this property isn't used anywhere yet, correct? Just trying to make sure I understand this correctly